backend/resume_evaluations/router.py

```
@@ -0,0 +1,19 @@
+    tags=["resume_evaluations"],
+)
+
+
+@router.post("/evaluate")
+def evaluate(payload: ResumeEvaluationIn) -> ResumeEvaluationOut:
+    """Evaluate a candidate"s resume against a job description."""
+    try:
+        return evaluate_resume(payload.resume_text, payload.job_description)
+    except ResumeEvaluationError as err:
+        # preserve the original traceback
+        raise HTTPException(status_code=500, detail=str(err)) from err
```

Pylint (3.3.7)

[error] Exception cause set to something which is not an exception, nor None

(E0705)